hacking/build_library/build_ansible/command_plugins/dump_keywords.py

```diff
@@ -0,0 +1,121 @@
+# coding: utf-8
+# Copyright: (c) 2019, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import importlib
+import os.path
+import pathlib
+import re
+from distutils.version import LooseVersion
+
+import jinja2
+import yaml
+from jinja2 import Environment, FileSystemLoader
+
+from ansible.module_utils._text import to_bytes
+
+# Pylint doesn't understand Python3 namespace modules.
+from ..change_detection import update_file_if_different  # pylint: disable=relative-beyond-top-level
+from ..commands import Command  # pylint: disable=relative-beyond-top-level
+
+
+DEFAULT_TEMPLATE_DIR = pathlib.Path(__file__).absolute().parents[4] / 'docs/templates'
```

Comment:
I believe more explicit top-level dirs would read better:
```suggestion
DEFAULT_TEMPLATE_DIR = (
    pathlib.Path(__file__) / '..' / '..' / '..' / '..' / 'docs/templates'
).absolute()
```

Reply:
I like it.  Merged thanks

Reply:
Note, this suggestion was invalid.  You can specify a path relative to a directory but not relative to a file.  changing it to take the first parent fixes that:
```
 DEFAULT_TEMPLATE_DIR = pathlib.Path(__file__).parent / '..'/ '..' / '..' / '..' / 'docs/templates' 
```

Once parent is added, though, the readability argument for using '..' goes away as you then have to both know why parent is being asked for and what the relative pathing is going to equate to.  Switching back to the original version.

Reply:
I'd probably still want to see `.absolute()` in the end

Reply:
Actually, after re-reading hte docs, I think that .absolute() should be changed to resolve() and stay where it is.

Reply:
well, maybe. I haven't checked what `.absolute()` does.